app/js/routes.jsx

```diff
@@ -7,16 +7,20 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react'
+import { ToastContainer } from 'react-toastify';
 import { Route, Switch } from 'react-router-dom';
 import App from './components/App';
 
 import Login from './components/login/Login';
 
 export default () => {
   return (
-    <Switch>
-      <Route exact path="/" component={App} />
-      <Route path="/login" component={Login} />
-    </Switch>
+    <div>
```

Comment:
@dondrzzy How about using `Fragment` instead of extra dom node(`div`)?

Reply:
Thanks, @muhozi, Fragments do improve overall application performance. Addressed.